src/Axes/cartesianAxes.ts

```diff
@@ -269,17 +270,33 @@ export class CartesianAxes extends Axes {
     const g = this.canvas.select('g.labels')
     if (this.options.y) {
       if (this.options.y[0]?.label) {
-        g.select('.y0.label')
-          .text(this.options.y[0].label)
+        if (this.options.y[0].labelOrientation === LabelOrientation.Vertical) {
+          const offset = this.options.y[0].labelOffset ?? 0
+          g.select('.y0.label')
+            .attr('x', -this.height / 2)
+            .attr('y', -30 - offset)
+            .text(this.options.y[0].label)
+        } else {
+          g.select('.y0.label')
+            .text(this.options.y[0].label)
```

Comment:
This part is also used in the 'if' case. Can it be taken outside of the if-else-statement?

Reply:
Moved code up 1 level